cmake/Buildblocktestcore.cmake

```diff
@@ -4,9 +4,8 @@
 
 include(YCMEPHelper)
 
-ycm_ep_helper(blockTest TYPE GIT
+ycm_ep_helper(blocktestcore TYPE GIT
               STYLE GITHUB
               REPOSITORY robotology/blocktest.git
               TAG master
-              COMPONENT core
```

Comment:
Please leave this line, as it is not related to the name of the CMake package at all.